test/integration/puppetlabs/services/certificate_authority/ca_true_test.clj

```diff
@@ -0,0 +1,34 @@
+(ns puppetlabs.services.certificate-authority.ca-true-test
+  (:require [clojure.test :refer :all]
+            [clojure.tools.logging :as log]
+            [puppetlabs.puppetserver.bootstrap-testutils :as bootstrap]
+            [puppetlabs.http.client.sync :as http-client]
+            [clojure.pprint :as pprint]
+  )
+)
+
+(deftest ca-true-test
+  (testing "We are validating that a csr with the CA:TRUE extension is rejected for signing by the server"
+      (bootstrap/with-puppetserver-running
+        app
+        {:jruby-puppet
+        {:master-conf-dir "./dev-resources/puppetlabs/ca_true_test/master/conf"}}
+        (let
+          [response (http-client/put (str "https://localhost:8140/"
+                                          "puppet-ca/v1/certificate_status/happy_distinguished_name")
+                                          {:ssl-cert    "./dev-resources/puppetlabs/ca_true_test/master/conf/ssl/ca/ca_crt.pem"
+                                           :ssl-key     "./dev-resources/puppetlabs/ca_true_test/master/conf/ssl/ca/ca_key.pem"
+                                           :ssl-ca-cert "./dev-resources/puppetlabs/ca_true_test/master/conf/ssl/ca/ca_crt.pem"
```

Comment:
nitpick: we have now agreed (after a lot of back-and-forth on the topic) to not align map values as part of our clojure style guide (https://github.com/puppetlabs/pl-clojure-style#other-style-considerations)